### PR TITLE
Spellchecker - Added reusable workflow and fixing spell mistake

### PR DESCRIPTION
**Pull request by ssarmokadam**
Adresses/Fixes #<ISSUENO>.

Implements

* 
* 
*

### CODE REVIEW

File: documentation/cobigen-openapiplugin.asciidoc [hunk 1]
```diff
@@ -40,18 +40,18 @@ Additionally, the java plugin provides the ability to match packages (`container
...
-Furthermore, it provides the ability to extract information from each input for further processing in the templates. The values assigned by variable assignments will be made available in template and the `destinationPath` of context.xml through the namespace `variables.<key>`. The OpenApi Plug-in currently provides two different mechanisms:
+Furthermore, it provides the ability to extract information from each input for further processing in the templates. The values assigned by variable assignments will be made available in template and the `destinationPath` of context.xml through the namespace `variables.<key>`. The OpenAPI Plug-in currently provides two different mechanisms:
 
 * type 'constant' -> constant parameter
 +
 [source,xml]
 ----
 <trigger id="..." type="openapi" templateFolder="...">
-    <containerMatcher type="element" value="OpenApiFile"/>
+    <containerMatcher type="element" value="OpenAPIFile"/>
```
Comment (maybeec) [inline documentation/cobigen-openapiplugin.asciidoc:54]: this is code, you should not change here anything, please check all occurrences

File: documentation/cobigen-openapiplugin.asciidoc [hunk 2]
```diff
@@ -40,18 +40,18 @@ Additionally, the java plugin provides the ability to match packages (`container
 
 * type 'element'
 
-The container matcher matches elements as Java Objects, in this case will be always an `OpenApiFile` object. (See link:cobigen-core_configuration#containermatcher-node[`containerMatcher` semantics] to get more information about `containerMatchers` itself.)
+The container matcher matches elements as Java Objects, in this case will be always an `OpenAPIFile` object. (See link:cobigen-core_configuration#containermatcher-node[`containerMatcher` semantics] to get more information about `containerMatchers` itself.)
 
-[[openapi-variable-assignment]]
+[[OpenAPI-variable-assignment]]
```
Comment (maybeec) [inline documentation/cobigen-openapiplugin.asciidoc:45]: this is a reference, you should not change here anything, please check all occurrences

File: documentation/howto-devonfw-adapt_template.asciidoc [hunk 3]
```diff
@@ -11,7 +11,7 @@ Right click in any part of the package explorer, then click on CobiGen -> Adapt
 
 image:images/howtos/e2e_gen/image56.png[]
 
-Click _Ok_:
+Click `_Ok_`:
```
Comment (maybeec) [inline documentation/howto-devonfw-adapt_template.asciidoc:14]: please remove underscores or simply correct to OK

File: documentation/howto_angular-client-generation.asciidoc [hunk 4]
```diff
@@ -194,6 +194,6 @@ http://localhost:4200
 
 image::images/howtos/angular4-gen/ng4gen_5.png[Running Angular 8 app,width="450"link="images/howtos/angular4-gen/ng4gen_5.png"]
 
-Once finished, the browser will open automatically at the previous localhost URL showing the Angular 8 application, using the credentials set at the devon4j java server you will be able to access.
+Once finished, the browser will open automatically at the previous local host URL showing the Angular 8 application, using the credentials set at the devon4j java server you will be able to access.
```
Comment (maybeec) [inline documentation/howto_angular-client-generation.asciidoc:197]: localhost is a technical term, should not be splitted

File: documentation/howto_devon4net.asciidoc [hunk 5]
```diff
@@ -88,15 +84,14 @@ In order to generate the files using the Cobigen CLI it is needed to do the foll
...
 ==== [navy]#Run the application#
 
-After the configuration of the database, open a terminal in path: *[Project_Name]/Devon4Net.Application.WebAPI* and then type:
+After the configuration of the database, open a terminal in path: `*[Project_Name]/Devon4Net.Application.WebAPI*` and then type:
 
-    dotnet run
+    `dotnet run`
     
-This will deploy our application in our localhost with the port 8082, so when you click https://localhost:8082/swagger/index.html[here] (https://localhost:8082/swagger) you can see, in swagger, all the services and the data model.
+This will deploy our application in our local host with the port 8082, so when you click https://localhost:8082/swagger/index.html[here] (https://localhost:8082/swagger) you can see, in swagger, all the services and the data model.
```
Comment (maybeec) [inline documentation/howto_devon4net.asciidoc:97]: please check all occurences

File: documentation/howto_ionic-client-generation.asciidoc [hunk 6]
```diff
@@ -6,7 +6,7 @@ toc::[]
 We are going to show you **how to** generate a CRUD Ionic application from an **ETO**
 using CobiGen. 
 
-NOTE: This is a short introduction to the Ionic generation. For a deeper tutorial including the generation of the backend, we strongly recommend you to follow link:files/HOW-TO-devonfw-ide-CobiGen-PoC-E2E_v3.2.pdf[this document].
+NOTE: This is a short introduction to the Ionic generation. For a deeper tutorial including the generation of the back end, we strongly recommend you to follow link:files/HOW-TO-devonfw-ide-CobiGen-PoC-E2E_v3.2.pdf[this document].
```
Comment (maybeec) [inline documentation/howto_ionic-client-generation.asciidoc:9]: backend is a technical term, please allow in dict. check all occurences

File: documentation/mgmt_ide-setup.asciidoc [hunk 7]
```diff
@@ -1,69 +1,69 @@
...
 
 == Eclipse Plugin Installation
 
-For some parts of Cobigen, you will have to have additional plugins installed.
+For some parts of CobiGen, you will have to have additional plugins installed.
 
 === Plugin development
 
-If you want to develop Cobigen plugins (openapiplugin, javaplugin etc.) you need to have the eclipse PDE plugin available. 
+If you want to develop CobiGen plugins (OpenAPI plugin, Java plugin etc.) you need to have the eclipse PDE plugin available.
```
Comment (maybeec) [inline documentation/mgmt_ide-setup.asciidoc:31]: please just allow "Plug-in" as term - plugin should not be valid to make it consistent

**Review by maybeec — CHANGES_REQUESTED** [review]

File: documentation/cobigen-openapiplugin.asciidoc (hunk 1 above)
Comment (ssarmokadam) [reply documentation/cobigen-openapiplugin.asciidoc]: reverted changes

File: documentation/cobigen-openapiplugin.asciidoc (hunk 2 above)
Comment (ssarmokadam) [reply documentation/cobigen-openapiplugin.asciidoc]: reverted changes in all occurences

File: documentation/howto_devon4net.asciidoc (hunk 5 above)
Comment (ssarmokadam) [reply documentation/howto_devon4net.asciidoc]: changed to localhost

File: documentation/howto_ionic-client-generation.asciidoc (hunk 6 above)
Comment (ssarmokadam) [reply documentation/howto_ionic-client-generation.asciidoc]: added to dict and done changes in documents

File: documentation/mgmt_ide-setup.asciidoc (hunk 7 above)
Comment (ssarmokadam) [reply documentation/mgmt_ide-setup.asciidoc:32]: I think, plugin is valid word and not detected as spell mistake. Also plugin is used at many occurrences... changing it at every place will take too much effort as we cannot use replace all blindly. What do you think can we keep plugin in document or should it be replaced by Plug-in?

**Comment by maybeec** [comment]
Thanks a lot @ssarmokadam! Great achievement

**Pull request merged**